app/controllers/application_controller.rb

```diff
@@ -2,4 +2,12 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def new_session_path(scope)
```

Comment:
Unused method argument - `scope`. If it's necessary, use `_` or `_scope` as an argument name to indicate that it won't be used. You can also write as `new_session_path(*)` if you want the method to accept any arguments but don't care about them.